contrib/contrib-vmss-publicipprefixinstances/README.MD

```diff
@@ -0,0 +1,45 @@
+# ARM CloudGuard IaaS Scale Set with instances' public IPs allocated from an existing public IP prefix
+
+This directory contains the CloudGuard IaaS VMSS template derived from the 20191003 vmss-v2 Marketplace scale set template.
+This template adds the ability to allocate IP addresses for instances from an existing public IP prefix.
+The VM scale set is deployed in the same existing resource group which includes the public IP prefix.
+
+## Changes from Marketplace template
+* added parameters:
+- existingPublicIPprefixGroupName
```

Comment:
these need to be double dashed to be nested. e.g.:
* added parameters:
-- existingPublicIPprefixGroupName